packages/framework/fluid-framework/src/index.ts

```diff
@@ -54,13 +54,26 @@ export type {
 	ReplaceIEventThisPlaceHolder,
 	FluidObject, // Linked in doc comment
 	FluidObjectProviderKeys, // Used by FluidObject
+	// The event APIs are known to conflict, and this is intended as the exports via `@fluidframework/core-interfaces` are preferred over the deprecated ones from `@fluidframework/tree`.
+	// eslint-disable-next-line import/export
+	Listeners,
+	// The event APIs are known to conflict, and this is intended as the exports via `@fluidframework/core-interfaces` are preferred over the deprecated ones from `@fluidframework/tree`.
```

Comment:
Duplicating this message isn't the most concise way to communicate this.

Might be cleaner to use a an `/* eslint-disable import/export */` then `/* eslint-enable  import/export`  ( https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1 ) and a single comment on the disable